public/js/actions/management.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+import $ from 'jquery';
+import * as actionTypes from 'constants/action-types';
+
+
+export function error(debugMessage) {
+  return {
+    type: actionTypes.APP_ERROR,
+    error: {debugMessage: debugMessage},
+  };
+}
+
+export function manageCards() {
+  return function(dispatch) {
+    $.ajax({
+      method: 'get',
+      url: '/api/braintree/mozilla/paymethod/',
+      context: this,
+    }).then(function(data) {
+      dispatch({
+        type: actionTypes.MANAGE_CARD_LIST,
```

Comment:
I think this might be less ambiguous if it were explicitly `GET_CARD_LIST` and `management.getCardList()`. The word manage could mean get or post. Also, I think we should get in the habit of naming things `GET_PAY_METHODS` because when we add bitcoin we'll have to refactor less things :)